neoexchange/core/plots.py

```diff
@@ -881,7 +888,7 @@ def phase_lc(lc_data, period, base_date):
 
 
 def translate_from_alcdef_filter(filt):
-    if filt not in ['B, V, R, I, J, H, K']:
+    if filt not in 'B, V, R, I, J, H, K':
```

Comment:
This works !? Would have thought this would need to be a list of string literals but python regularly surprises

Reply:
This works. A list of string literals would also work. But definitely not whatever mess I had in there originally.